nixos/doc/manual/release-notes/rl-1703.xml

```diff
@@ -88,6 +90,12 @@ following incompatible changes:</para>
      <literal>networking.timeServers</literal>.
    </para>
   </listitem>
+
+  <listitem>
+    <para><literal>pkgs.overridePackages</literal> function no longer exists.
+     Instead import Nixpkgs a second time using <literal>import pkgs.path {
+    overlays = [ <replaceable>...</replaceable> ]; }</literal>.</para>
```

Comment:
This is not clear what `pkgs` is and what would be the contents of `overlays`. It's better to point to overlays section in the manual.